taskfiles/Kind.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3'
```

Comment:
do we need this one ? (i know i also added it in nic-feature-discovery but been thinking... )

with minikube just need to point docker to the same docker daemon.

Reply:
sometimes I'm using Kind instead of minikube. Let's keep it for now